pkg/cli/dump_test.go

```diff
@@ -490,3 +490,30 @@ ALTER TABLE orders VALIDATE CONSTRAINT fk_customer;
 		t.Fatalf("expected: %s\ngot: %s", want1, dump1)
 	}
 }
+
+func TestDumpingUUIDColumns(t *testing.T) {
+	// Testing use case described in #33664, that's it
+	// to create a table with UUID generated columns and
+	// then insert few rows. Dump table with the content
+	// and assert it can do it successfully.
```

Comment:
place the comment above the function so that the defer is the first line in the body (required by the linter)